Sources/Turf/JSON.swift

```diff
@@ -209,3 +217,39 @@ extension JSONValue: Codable {
         }
     }
 }
+
+/**
+ A type that can be represented as a `JSONValue` instance.
+ */
+public protocol JSONValueConvertible {
```

Comment:
Should `Optional` be `JSONValueConvertible` in situations where its generic type is too?